docs/evidence/issue-113/count-treatments.html

```
@@ -0,0 +1,383 @@
+<title>Groups, Pieces, Proposals</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)